src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -3448,6 +3448,7 @@ def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=
                      no_wait=False,
                      workspace_id=None,
                      customer_admin_group_id=None):
+    logger.warning('Support for the creation of ARO 3.11 clusters ends 30 Nov 2020. Please see aka.ms/aro/4 for information on switching to ARO 4.')  # pylint: disable=line-too-long
```

Comment:
can you pls also add this in _help.py? thanks.

Reply:
@yungezz good call, done - thanks